nixos/modules/services/x11/display-managers/gdm.nix

```diff
@@ -142,8 +142,6 @@ in
           GDM_X_SERVER_EXTRA_ARGS = toString
             (filter (arg: arg != "-terminate") cfg.xserverArgs);
           XDG_DATA_DIRS = "${cfg.session.desktops}/share/";
```

Comment:
This line also seems like it should be redundant now?

Reply:
I don't think `session.desktops` is going to be installed/linked.